Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -539,23 +531,6 @@ private async Task UpdateActiveFilterIndex( int activeItemIndex )
                 SelectedText = GetItemText( item );
                 await SelectedTextChanged.InvokeAsync( SelectedText );
             }
-
-            if ( ActiveItemIndex >= 0 )
-            {
-                // ensure item in view
-                var listInfo = await JSUtilitiesModule.GetElementInfo( ElementRef, null );
-                var itemInfo = await JSUtilitiesModule.GetElementInfo( null, GetDropdownItemElementId( ActiveItemIndex ) );
-
-                if ( itemInfo.OffsetTop < listInfo.ScrollTop || itemInfo.ClientHeight > listInfo.ClientHeight )
-                {
-                    await JSUtilitiesModule.SetProperty( ElementRef, "scrollTop", itemInfo.OffsetTop );
-                }
-
-                if ( itemInfo.OffsetTop + itemInfo.OffsetHeight > listInfo.ScrollTop + listInfo.ClientHeight )
-                {
-                    await JSUtilitiesModule.SetProperty( ElementRef, "scrollTop", itemInfo.OffsetTop + itemInfo.OffsetHeight - listInfo.ClientHeight );
-                }
-            }
```

Comment:
Ok, so you've reverted this, since we had this already implemented up/down support on another PR.

You've seem to have detected a bug there, and if for some reason the implementation there does not work fully, we might borrow something from your implementation. Only thing to note is that there seems to be too much Javascript interop at once, interop is costly, and if you can do all the operations in one interop it's ideal.

Reply:
> You've seem to have detected a bug there

Mitigated it with Task.Yield, now it scrolls up fine. Any idea what it means?

<img width="585" alt="image" src="https://user-images.githubusercontent.com/22550674/180627117-8dc9dada-d7b7-42ab-b6ca-8565986fadc0.png">

However this scrolling is kinda slow compared to my scrolling which is similar to listbox scrolling. Obviously scrollIntoView is better than my custom arithmetic, maybe there's a way to improve perf. 

Also, just for fun, try running Narrator and see how it sees the structure of the component and how it allows keyboard navigation through checkboxes (but not items).. In accessibility, there's a tree similar to DOM, which describes the structure of the application for screen readers. So ideally the components should be made such that their structure makes sense to accessibility. Just play with Narrator to get a different perspective when you have a minute...

Reply:
Using element.scrollTo() and calculating it ourselves does not have that bug. So I removed Yield() and implemented it with scrollTo() in JS..